libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -207,14 +304,33 @@ class AP_Airspeed
         float last_saved_ratio;
         uint8_t counter;
     } state[AIRSPEED_MAX_SENSORS];
-
+    
+    struct {
+        float raw_airspeed;
+        float airspeed;
+        float last_pressure;
+        float filtered_pressure;
+        float corrected_pressure;
+        float EAS2TAS;
+        float temperature;
+        float ratio;
+        float offset;
+        
+        // Blended airspeed
+        AP_Int8 enable;
+        
+        bool blend_active;
```

Comment:
rename to "active" if it's in the blend_state already.

Reply:
OK